MathlibTest/Simps.lean

```diff
@@ -110,7 +110,7 @@ structure Equiv' (α : Sort _) (β : Sort _) where
 infix:25 (priority := default+1) " ≃ " => Equiv'
 
 /- Since `prod` and `PProd` are a special case for `@[simps]`, we define a new structure to test
```

Comment:
This should be /--, right?

Reply:
This is a test file and there are several other declarations that have such a "doc-string".  I would not change just this one.